src/app/workflow/versions/versions.component.scss

```diff
@@ -49,3 +49,14 @@ tr.mat-row.highlight {
 tr.highlight > .mat-cell:first-of-type > span {
   padding-left: 5px;
 }
+
+.mat-cell:not(:first-of-type),
+.mat-header-cell:not(:first-of-type) {
+  padding-left: 1rem;
```

Comment:
Would recommend adding a comment about why these styles override the default https://github.com/dockstore/dockstore-ui2/blob/f145992b8085007f0c52b1c010dbaa7d1103ef27/src/styles.scss#L1249-L1259